website/public/logo.svg

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Maybe we could move this to a `res/` folder with all the other images

Reply:
the favicon should be in the public folder -- read this in some documentation for react, don't remember where